### PR TITLE
[SIG-2464] Configure Weesp Logo

### PR DESCRIPTION
**Pull request by jpoppe**

### CODE REVIEW

File: acc.config.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-    "height": 0,
-    "smallWidth": 0,
-    "smallHeight": 0
+    "url": "/assets/logos/logo-weesp.svg",
+    "width": "unset",
+    "height": "100%",
+    "maxHeight": "67px",
+    "smallWidth": "unset",
+    "smallHeight": "100%",
+    "maxSmallHeight": "67px"
```
Comment (janjaap) [inline acc.config.json]: Can't we solve all this with CSS? If we start going down the route of having CSS rule values in our configuration, then it'll become a mess fairly quickly.

Comment (jpoppe) [reply acc.config.json]: Probably we can, however I 'continued' on the work I merged and this is IMHO out of scope for SIG-2464.

Comment (janjaap) [reply acc.config.json]: The scope of SIG-2464 is the logo, so I wouldn't say it's out of scope. Let's try to solve this CSS-side to limit the amount of config lines required for just an image.

Comment (jpoppe) [reply acc.config.json]: I totally agree, will put this on resolved since we agreed to file a new ticket for this task,

**Review by aditudorache — APPROVED** [review]

**Review by janjaap — APPROVED** [review]

**Pull request merged**